### ISSUE TITLE
System tracking light/dark mode not working

### ISSUE THREAD

**Issue by gingerbeardman** [jcsalterego/Sky.app#29]
The website https://bsky.app works correctly and changes colour with light/dark mode.

But the Sky app does not.

If I go into settings and click "system" = no change
If I change my mode to dark whilst the app is open = no change
And relaunching the app after a mode switch = no change 

Related: #16

**Comment by jcsalterego** [comment]
Yes, I think this is a problem. The sync actually matches what's in the app with the window chrome. I think the part that's broken is what the app is telling the website what scheme is the OS is in

**Comment by gingerbeardman** [comment]
Yes! That makes sense.

**Comment by jcsalterego** [comment]
Working on it... https://github.com/jcsalterego/Sky.app/compare/dark-mode-system-sync

**Comment by jcsalterego** [comment]
This should be much better in 0.4.2, please give it a shot.

What is NOT supported at this time: #37

**Comment by gingerbeardman** [comment]
I'll use it! thanks
